shoop/front/templates/shoop/front/product/detail.jinja

```diff
@@ -130,10 +130,10 @@
         </div>
     {% endif %}
 
-    {% set related_products = shoop.product.get_products_bought_with(product) %}
+    {% set related_products = shoop.product.get_product_cross_sells(product) %}
     {% if related_products %}
```

Comment:
134..144, 147..157 and 160..170 are identical aside from a title; make a macro out of them.